sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizeHealthcareEntitiesTests.cs

```diff
@@ -143,9 +141,9 @@ public async Task RecognizeHealthcareEntitiesWithSkipParameter()
 
             await operation.WaitForCompletionAsync(PollingInterval);
 
-            RecognizeHealthcareEntitiesResultCollection resultCollection = operation.Value;
+            AnalyzeHealthcareEntitiesResultCollection resultCollection = operation.Value;
 
-            Assert.IsNotNull(resultCollection[0].TextAnalyticsError);
+            Assert.IsNotNull(resultCollection[0].HasError);
```

Comment:
`HasError` https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsResult.cs#L48 is bool so it can never be null. 

u could either check the value is false, or use `Error` instead of `TextAnalyticsError`